app/views/static_pages/home.html.erb

```diff
@@ -1,37 +1,33 @@
-<div class="content">
-  <h2 class="text-center">A community for petites, by petites!</h2>
-  <div class="row">
-    <div class="col-md-2">
-    </div>
+<div class="row spacer-row text-center">
+  <%= image_tag('tpl-logo-black-446x250.png', class: 'center-block img-responsive') %>
+  <h2>A community for petites, by petites!</h2>
```

Comment:
I'd love to see more spaced out & sexy styling on this section; the content is great for now, but just a thought as you add more polish.